SPECS/usbredir/usbredir.spec

```diff
@@ -70,14 +71,15 @@ A simple USB host TCP server, using libusbredirhost.
 %{_libdir}/libusbredir*.so
 %{_libdir}/pkgconfig/libusbredir*.pc
 
-%files server
+%files tools
 %license COPYING
 %{_bindir}/usbredirect
-%{_sbindir}/usbredirserver
 %{_mandir}/man1/usbredirect.1*
-%{_mandir}/man1/usbredirserver.1*
 
 %changelog
+* Tue Jan 30 2024 Dallas Delaney <dadelan@microsoft.com> - 0.13.0-1
+- Upgrade to 0.13.0
```

Comment:
it looks like you also renamed the "server" subpackage to "tools". Can you also mention this in the changelog

Reply:
changelog is now updated with subpackage change